ci/benchmarks/partial-conv/evo2_pretrain.yaml

```diff
@@ -54,10 +69,10 @@ script: |-
   --max-steps=${max_steps} \
   --limit-val-batches=20 \
   --log-every-n-steps=50 \
-  --val-check-interval=500 \
+  --val-check-interval=${max_steps} \
```

Comment:
it would be good to periodically run validation, at least in the partial conv run. That way we also test that validation still works and checkpointing still works. Sometimes those things break.

Reply:
ok, great, I will run it every 50 or 500 steps?